docs/swagger/swagger.yaml

```diff
@@ -363,12 +402,12 @@ components:
             - OTHER
         images:
           type: array
-          required:
-            - url
-            - signature
-            - owner
           items:
             type: object
+            required:
+              - url
+              - signature
```

Comment:
NO please do NOT require signature

Reply:
This is the old endpoint. We'll need to remove it from the endpoind validation and add logic to calculate it, but we don't have to do so bc we're going to migrate to this new endpoint anyway.